cli/gitleaks/__init__.py

```diff
@@ -0,0 +1,79 @@
+#
+# Copyright (C) 2023 Red Hat, Inc.
+#
+#    This program is free software: you can redistribute it and/or modify
+#    it under the terms of the GNU General Public License as published by
+#    the Free Software Foundation, either version 3 of the License, or
+#    (at your option) any later version.
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU General Public License for more details.
+#
+#    You should have received a copy of the GNU General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+"""Module building gitleaks cli command"""
+from typing import Any
+
+import click
+
+from cli.gitleaks.gitleaks import GitleaksConfig
+
+DEFAULT_SERVER_URL = "https://patterns.security.redhat.com"
```

Comment:
isn't `"https://patterns.security.redhat.com"` internal url?
should we expose it in the code?

Reply:
I will make this change and expect the environment variable to already be set. Should we also remove the urls for the Jira staging environment?